src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -141,7 +139,7 @@ export default class DefaultAudioVideoController
     webSocketAdapter: WebSocketAdapter,
     mediaStreamBroker: MediaStreamBroker,
     reconnectController: ReconnectController,
-    eventReporter?: EventReporter
+    eventController?: EventController
```

Comment:
Why do we need it to always be able to be published? We currently don't have any part of the SDK that relies on events being published, obviously it wouldn't be able to publish events without it, but the SDK will work fine without it so I'm not sure why it needs to be a hard dependency.

Reply:
Good point. So it should not be optional in device controller either?

Reply:
> Why do we need it to always be able to be published? We currently don't have any part of the SDK that relies on events being published.

Yes, we need always publish for the ingestion service and SDK is not dependent but the ingestion service is. I meant that we have to publish meeting events by default and thats a must for reporting events out currently for event ingestion so any benefits in keeping it optional? Generally it would be good to avoid optional parameters on methods. I am ok with keeping it optional.